test/features/qdevice_usercase.feature

```diff
@@ -0,0 +1,62 @@
+@qdevice
+Feature: Verify usercase master survive when split-brain
+
+  According to https://confluence.suse.com/x/SoaHFQ:
```

Comment:
This url is private, right?
Should it go here?

Reply:
OK, changed